tests/fail/elaboration/implicit-args/app-plicity-mismatch.snap

```diff
@@ -0,0 +1,19 @@
+stdout = ''
+stderr = '''
+error: implicit argument was applied to an explicit function
+  ┌─ tests/fail/elaboration/implicit-args/app-plicity-mismatch.fathom:6:21
+  │
+6 │ let _ : Bool = id1 @Bool true;
+  │                ---  ^^^^ implicit argument
+  │                │     
+  │                explicit function of type fun (A : Type) -> A -> A
```

Comment:
How does this error read when applying an implicit arg to something that isn’t a function? Like `true @Bool`

Reply:
Same as an explicit argument application